lib/src/lints/function_lines_of_code/models/function_lines_of_code_parameters.dart

```diff
@@ -5,16 +5,22 @@ class FunctionLinesOfCodeParameters {
   /// exceeding this limit triggers a warning.
   final int maxLines;
 
+  /// Method names to be excluded from the rule check
```

Comment:
```suggestion
  /// Function names to be excluded from the rule check
```
Does it make sense to be consistent with all the rest documentation/naming?